public/src/canadian-championship-bidding-policy.md

```diff
@@ -0,0 +1,73 @@
+- The Canadian Championship Selection period _should_ be a minimum of 14 days.
+- The selection of a host for the Canadian Championships _should_ be concluded at least 9 months before the start of the event.
+
+## Canadian Championship Application Submission Process
+
+- A Canadian Championship Application will consist of a PDF document that contains information to support the Canadian Championship Selection Committee with the Canadian Championship Selection Process.
+- Every application **must** have a point person if an application is being submitted on behalf of a larger organization team.
+  - All communications regarding the application _should_ be directed to the point person of an application.
+  - The point person for a Canadian Championship Application **must** be a member of Speedcubing Canada.
+- Every application **must** include the following noted in this list:
+  - Location and Time
+    - Possible dates
+    - Venue 
+      - The organization team **must** only have one venue being proposed in an application. Additional possible venues _may_ be added to an appendix in the application.  
+    - City/Town of Venue
+    - Number of days for the competition
+  - Organization Team
+    - Core Team Members
+    - Experience of Core Team
+  - Budget
+    - Itemized budget that includes Cost of Venue, Proposed Registration Cost, Prizes, Miscellaneous Equipment Costs, and other expenses relevant to a WCA competition
+    - Additional equipment requirements outside of the Organization Team (ex. Timers/Displays, Banners)
+- Every application _may_ include the following:
+  - Ease of national travel to the location
+  - Availability of lodging and accommodation
+  - Tourism in location
+- Applications may include additional information that may be important in the decision-making process for the Canadian Championship Selection Committee
+
+## Canadian Championship Selection Process
+
+- Once the Application Submission Process is complete, the Canadian Championship Selection Process will begin.
+- The review process **must** consist of communications between the Canadian Championship Selection Committee and a final vote to select the host of the Canadian Championship.
+  - The review period _may_ include meetings between the Canadian Championship Selection Committee.
+  - Meetings _may_ be proposed with each organization team before voting has begun.
+    - Modifications _may_ be made to an application if it is required by the Canadian Championship Selection Committee.
+- The voting process **must** be conducted using a ranked ballot system.
+  - Each member of the Canadian Championship Selection Committee will receive a ballot listing all submitted applications.
+  - Committee members *must* rank the bids in order of preference, assigning a unique rank to each bid.
+    - 1 will be considered the highest preference, with each descending number representing a lower preference.
```

Comment:
Does this type of detail belong in this document? How the ranked ballot system implements these requirements should be up irrelevant here?

Reply:
Yeah, good point. I'll go ahead and acknowledge that we have a voting process, but the specifics aren't needed.

Reply:
I just meant the fact that 1 is considered highest is an implementation detail that doesn't matter. I liked all the other points.